notebooks/community/model_garden/model_garden_tfvision_image_classification.ipynb

```diff
@@ -0,0 +1,771 @@
+{
```

Comment:
<p>seems redundant, you already have 'endpoint'</p><p> </p><p>what about delete the model?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/1625/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='22'/>

Reply:
<p>done</p>